build/tools/ExtractRuleDescFromJson/PluginRule.cs

```diff
@@ -18,21 +18,26 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System.Text.Json.Serialization;
-
 namespace ExtractRuleDescFromJson;
 
-public class Rule
+/// <summary>
+/// Data-class: rule properties, as defined by the plugin API
+/// </summary>
+public class PluginRule
```

Comment:
This class is only used in the generator tool, so I made the properties get-set so I didn't have to have a constructor.

I also changed the name from `Rule` to make it clear that this the version of rule data that we get from the Sonar analyzer plugin api.